docs/eventing/samples/apache-rocketmq-source/README.md

```diff
@@ -0,0 +1,199 @@
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: apacherocketmqsource
+  spec:
+      topic: test
+      namesrvAddr: xx.xx.xx.xx:9876 # nameserAddr of RocketMQ Cluster
+      groupName: groupname
+      sink:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: event-display
+   ```
+
+2. Deploy the event source.
+
+   ```
+   $ kubectl apply -f apacherocketmqsource.yaml
+      rocketmqsource.sources.eventing.knative.dev/apacherocketmqsource created
+   ```
+1. Check that the event source pod is running. The pod name will be prefixed with `rocketmq`.
+
+   ```
+   $ kubectl get pods
+      NAME                                                  READY     STATUS    RESTARTS   AGE
+      rocketmq-apacherocketmqsource-p7tdw-67f468884b-n4b9j   1/1       Running   0          4s
+   ```
+1. Ensure the RocketMQ event source started with the necessary
+   configuration.
+   ```
+   $ kubectl logs --selector='knative-eventing-source-name=apacherocketmqsource'
+    {"level":"info","ts":"2020-03-13T05:55:31.155Z","caller":"receive_adapter/main.go:80","msg":"Starting RocketMQ Receive Adapter. %v","adapter: ":{"K8sClient":{"LegacyPrefix":"/api"},"Namespace":"default","SecretName":"","SecretKey":"","Topic":"test2","NamesrvAddr":"101.200.125.226:9876","RNamespace":"","GroupName":"group2","SubscriptionID":"knative-eventing-default","SinkURI":"http://event-display2.default.svc/","AdCode":""}}
+time="2020-03-13T05:55:31Z" level=info msg="the consumer start beginning" consumerGroup=group2 messageModel=Clustering unitMode=false
+   ```
+
+## Verify
+
+1. Produce a message with [CLI Admin Tool](http://rocketmq.apache.org/docs/cli-admin-tool/) of Apache RocketMQ:
+
+   ```
+   $ sh bin/mqadmin sendMessage -t test2 -k 123 -p "hello" -n 127.0.0.1:9876
+      #Broker Name                      #QID  #Send Result            #MsgId
+      iZ2ze56wd7v87vxqhdd3j6Z           0     SEND_OK                 7B3945ED000027C170F040CF8EBB0000
+   ```
+1. Check that the RocketMQSource consumed the message and sent it to
+   its sink properly.
+
+   ```
+   $ kubectl logs --selector='knative-eventing-source-name= apacherocketmqsource'
+      subscribe callback: [Message=[topic=test2, body=hello, Flag=0, properties=map[CLUSTER:DefaultCluster CONSUME_START_TIME:1584080251333 KEYS:123 MAX_OFFSET:9 MIN_OFFSET:0 UNIQ_KEY:7B3945ED000027C170F040DD65BE0000 WAIT:true], TransactionId=], MsgId=AC14000100002A9F0000000000028A1E, QueueId=3, StoreSize=185, QueueOffset=8, SysFlag=0, BornTimestamp=1584080251327, BornHost=172.20.0.1:35518, StoreTimestamp=1584080251331, StoreHost=172.20.0.1:10911, CommitLogOffset=166430, BodyCRC=907060870, ReconsumeTimes=0, PreparedTransactionOffset=0]
+   ```
+
+1. Ensure the event display received the message.
+
+   ```
+   $ kubectl logs --selector='serving.knative.dev/service=event-display2' -c user-container
+
+  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 0.2
```

Comment:
we've been lately using v1.0. The 0.2 spec won't be supported by core eventing, so it might be worth updating this as well. You can take a look at examples in eventing or eventing-contrib. For example, https://github.com/knative/eventing/blob/7808608f380596d212104097c3e4639dbf4eb724/pkg/adapter/ping/adapter.go#L105

Reply:
Ok， I will update to v1.0.